src/educator/Main.hs

```diff
@@ -13,7 +13,7 @@ import Dscp.Educator (EducatorParams (..), educatorParamsParser, launchEducatorR
 main :: IO ()
 main = do
     educatorParams <- getEducatorParams
-    launchEducatorRealMode educatorParams $
+    launchEducatorRealMode educatorParams $ do
```

Comment:
Kind of unnecessary change :thinking:

Reply:
I looked to place to initialize schema, and my first intent was to place it here; however, I found a proper place in short time, but left this change out. Should I `git rebase -i` it out?